WalletWasabi.Gui/Tabs/EncryptionManager/VerifyMessageView.xaml

```diff
@@ -0,0 +1,27 @@
+﻿<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:controls="clr-namespace:WalletWasabi.Gui.Controls;assembly=WalletWasabi.Gui" Design.Width="600">
+  <controls:GroupBox Title="{Binding Title}" BorderThickness="0" Classes="docTabContainer">
+    <Grid Classes="content" Margin="0 40 0 0">
+      <Grid.RowDefinitions>
+        <RowDefinition Height="*" />
+        <RowDefinition Height="Auto" />
+        <RowDefinition Height="*" />
+        <RowDefinition Height="40" />
+        <RowDefinition Height="Auto" />
+      </Grid.RowDefinitions>
+      <controls:ExtendedTextBox Grid.Row="0" Text="{Binding Message}" Watermark="Message" UseFloatingWatermark="True" TextWrapping="Wrap" AcceptsReturn="True" />
+      <controls:ExtendedTextBox Grid.Row="1" Text="{Binding Address}" Watermark="Address" UseFloatingWatermark="True" />
+      <controls:ExtendedTextBox Grid.Row="2" Text="{Binding Signature}" Watermark="Signature" TextWrapping="Wrap" UseFloatingWatermark="True" />
+      <StackPanel Grid.Row="3"  Orientation="Horizontal" HorizontalAlignment="Right" IsVisible="{Binding IsVerified}" Spacing="10" Margin="10">
+        <Border Background="Transparent" ToolTip.Tip="Signature verified successfully!">
+          <Path Data="F1 M 23.7501,33.25L 34.8334,44.3333L 52.2499,22.1668L 56.9999,26.9168L 34.8334,53.8333L 19.0001,38L 23.7501,33.25 Z" Fill="#22B14C" Height="16" Width="16" Stretch="Fill" />
+        </Border>
+        <TextBlock Text="Signature verified successfully!" />
```

Comment:
ok, this can be an idiotic comment but I think the `"Signature verified successfully"` should be changed by something like `"Message authenticity verified"` because digital signatures are for preventing message forgery so, we are verifying the message was not modified and was send by a person that is who claims to be.

If that's what everybody means with `"Message authenticity verified"` then it is okay.